.eslintignore

```diff
@@ -1,26 +1,26 @@
 # unconventional js
-packages/*/blueprints/*/*files/
-vendor/
+/packages/*/blueprints/*/*files/
+/packages/*/vendor/
 
 # compiled output
+/packages/*/dist/
+/packages/*/tmp/
 /packages/-ember-data/docs/
-dist
-tmp
 
 # dependencies
-bower_components
-node_modules
+/packages/*/bower_components/
+/packages/*/node_modules/
```

Comment:
node_modules is usually created in the root directory.

Reply:
Nice catch!

Reply:
Updated pattern to include `node_modules` and `bower_components` in the root directory